### PR TITLE
Echo filenames as test data is being loaded

### PR DESCRIPTION
**Pull request by benjamingeer**
Because of a recent update to the RDF4J console, when a script such as `graphdb-se-local-init-knora-test.sh` is loading Knora test data, the name of each TTL file is no longer echoed to the terminal. This changes those scripts to use an [expect](https://linux.die.net/man/1/expecthttps://linux.die.net/man/1/expect) script that echoes each command that it sends to the RDF4J console, thus imitating the old behaviour of the console.

### CODE REVIEW

File: webapi/scripts/Readme.md [hunk 1]
```diff
@@ -8,33 +8,33 @@ This script will create the ``knora-test-unit`` repository and not load any data
...
  - with GraphDB running locally, e.g., graphdb distribution:
  ```$ graphdb-se-local-init-knora-prod.sh ```
- 
+
 
 ## Fuseki
 
-Nothing neede to do. Simply run Fuseki, either in a docker container or locally by using the supplied distribution.
\ No newline at end of file
+Nothing neede to do. Simply run Fuseki, either in a docker container or locally by using the supplied distribution.
```
Comment (tobiasschweizer) [inline webapi/scripts/Readme.md]: Do we still support Fuseki?

Comment (tobiasschweizer) [reply webapi/scripts/Readme.md]: typo "neede"

Comment (benjamingeer) [reply webapi/scripts/Readme.md]: All the Fuseki support is still there, it's just deprecated, like the docs say: http://www.knora.org/documentation/manual/rst/knora-api-server/deployment/webapi-getting-started.html?highlight=deprecated

**Comment by benjamingeer** [comment]
> Don't you want to do the same for `scripts/graphdb-se-knora-prod-data.ttl`?

OK, I will.

**Comment by tobiasschweizer** [comment]
> OK, I will.

So you will want it. Will you also do it ;-)?

**Comment by tobiasschweizer** [comment]
just joking ...

**Comment by benjamingeer** [comment]
> So you will want it. Will you also do it ;-)?

I did it even though I didn't want to. :)

OK now?

File: webapi/scripts/graphdb-se-local-init-knora-test.sh [hunk 2]
```diff
@@ -24,9 +24,9 @@ curl -X POST -H "Content-Type:application/x-turtle" -d "<http://www.knora.org/co
 
 printf "${GREEN}Repository created.\n\n${DELIMITER}Loading test data${NO_COLOUR}\n\n"
 
-cat graphdb-knora-test-data.ttl | $CONSOLE
+./graphdb-knora-test-data.expect $GRAPHDB
 
-printf "\n${GREEN}${DELIMITER}Creating Lucene index${NO_COLOUR}\n\n"
+printf "\n\n${GREEN}${DELIMITER}Creating Lucene index${NO_COLOUR}\n\n"
```
Comment (tobiasschweizer) [inline webapi/scripts/graphdb-se-local-init-knora-test.sh]: DO we need this extra leading newline?

**Pull request merged**